src/styles/components/_related-components.scss

```diff
@@ -1,10 +1,11 @@
 $breakpoint-tablet: 900px;
 
 .rel-component {
```

Comment:
I think we should discuss in a slack message. I'll message the apprentices and ask. 

I am not sure if this is in scope for this PR though as it would have to be changed on several pages. Is this something you or Jared can fix on your "style-tweaking" PR? 

@dustin-jw what do you think? Thanks! :)

Reply:
I'll add "renaming breakpoint variables" to my list of cards to write 😄

Reply:
Great, thank you! I'll leave the breakpoints as is for this card then. :)